src/modules/esl-utils/dom/traversing.ts

```diff
@@ -1,3 +1,4 @@
+import {isElement} from './api';
```

Comment:
Replace import with a more specific one. Ideally the consumers should have tree-shaking but we should not rely on it